docs/proposals/suggestion.md

```diff
@@ -21,6 +21,8 @@ Table of Contents
          * [BOHB](#bohb)
          * [TPE](#tpe)
          * [SMAC](#smac)
+         * [CMA-ES](#cma-es)
+         * [Sobol](#sobol)
```

Comment:
I think we should have the table somewhere in the docs with all supported Frameworks and Algorithms that these Suggestions support. Eventually, we don't need to modify the proposals docs.
WDYT @gaocegege @c-bata @johnugeorge ?
We can do it in the following PR.

Reply:
> I think we should have the table somewhere in the docs with all supported Frameworks and Algorithms that these Suggestions support. Eventually, we don't need to modify the proposals docs.

Sounds good! Should I revert the changes of `suggestion.md`?

Reply:
I think we can leave this change in `suggestion.md` for now.
That will simplify the process for us to create the doc table.